### PR TITLE
fix: add fees in TT price calculation

### LINKED ISSUE #1989: bug: Calculate Tranche Token Prices with `NavFees`
### Description
Current implementation in the pallet-pool-system for `trait TrancheTokenPrice` does not take into account accrued fees, as the epoch execution does. 

We need to fix that by adding the same flow as for epoch-closing.

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description
See issue

Fixes #1989

## Changes and Descriptions
* just adds fees to tranche token price calculation
 
# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: pallets/pool-system/src/impls.rs [hunk 1]
```diff
@@ -66,13 +66,15 @@ impl<T: Config> TrancheTokenPrice<T::AccountId, T::CurrencyId> for Pallet<T> {
...
-		// expensive
-		let (nav, nav_last_updated) = T::AssetsUnderManagementNAV::nav(pool_id)?;
-		let total_assets = pool.reserve.total.ensure_add(nav).ok()?;
+		let (nav_loans, nav_loans_updates) = T::AssetsUnderManagementNAV::nav(pool_id)?;
+		let (nav_fees, nav_fees_updated) = T::PoolFeesNAV::nav(pool_id)?;
+
+		let nav = Nav::new(nav_loans, nav_fees);
+		let total_assets = nav
+			.total(pool.reserve.total)
+			.unwrap_or(<T as Config>::Balance::zero());
```
Comment (lemunozm) [inline pallets/pool-system/src/impls.rs:77]: Super NIT: `unwrap_or_default()`

**Review by lemunozm — APPROVED** [review]
Makes sense! LGTM

File: pallets/pool-system/src/impls.rs (hunk 1 above)
Comment (mustermeiszer) [reply pallets/pool-system/src/impls.rs:77]: I am not the biggest fan of default in these case. Zero is really explicit and also what we want.

**Comment by mustermeiszer** [comment]
Why did the pipeline fcancel.

**Pull request merged**